x-pack/plugin/vector-tile/src/javaRestTest/java/org/elasticsearch/xpack/vectortile/VectorTileRestIT.java

```diff
@@ -896,8 +896,11 @@ public void testMultiValueAgg() throws Exception {
                 "percentilesAgg": {
                   "percentiles": {
                      "field": "value1",
-                     "percents": [95, 99, 99.9]
-                    }
+                     "percents": [95, 99, 99.9],
+                     "tdigest": {
+                       "compression": 10
```

Comment:
Why is compression 10 necessary here?

Reply:
This was the test OOMing with MergingDigest. I implemented a hybrid digest that should address the memory allocation issue, reverting this.